gazebo_world_ws/src/gazebo_launch/launch/gazebo.launch.py

```diff
@@ -13,6 +13,11 @@
 from ament_index_python.packages import get_package_share_directory
 
 ARGUMENTS = [
+    DeclareLaunchArgument(
+        "launch_gzclient",
+        default_value="False",
+        description="Launch gzclient, by default is False, which means headless mode",
+    ),
```

Comment:
This is a nice feature to have. However, should the default value be `True`?

Reply:
This decision is based on my recent experience. Since it’s difficult to get the information I need directly from `Gazebo`, instead of letting `Gazebo` launch its GUI, which clutters the screen and consumes computational resources, it’s more useful and helpful to open tools like `rqt` or `rviz` to monitor the data directly. The GUI from `Gazebo` may only needed for final demonstration or in some special cases.

However, we can set it to `"True"` by default and adjust it during development as needed. I don’t think this will have a significant impact!

Reply:
I'm not sure why `Gazebo` is so lag even when opening a small environment. But I still think it's better to set it to `True` by default.

We can explicitly disable the GUI in the README command instead. Thanks!

Reply:
If it’s a small environment, it shouldn’t lag much. Even when opening medium worlds like AWS house or hospital, there should only be a slight lag when loading. Only `citysim` might cause noticeable lag. Could you let me know how much resource Gazebo uses on your computer?

When I previously used the VLP-16 in Gazebo simulations, the lack of GPU acceleration caused noticeable lag. If the issue is significant, I might need to identify which model is causing the high resource cost during the simulation. Or are you currently using WSL? I also noticed a significant lag when I used WSL in the past.

Reply:
I'm not sure why `aws_small_house` lags a lot on my machine. All other environments seem fine...

This may be an issue on my side. I'll change the default to enabling GUI and merge this PR, thanks!

Reply:
We can continue discussing the performance issue if it becomes a problem in the future.